README.md

```diff
@@ -588,6 +588,18 @@ end
 
 Note though that if a dynamic with the same name exists in your `templates/dynamics/` directory it will get loaded since it has higher precedence.
 
+Templates can be also loaded from sparkle packs by defining `sparkle_pack_template`. No extension should be added in this case:
```

Comment:
The idea that the template name is the same as path is just a common way we do it. This doesn't have to be true, and you can have multiple templates per single file. For example you can have a file called `foo.rb` containing:
```
SparkleFormation.new(:foo1) do ... end
SparkleFormation.new(:foo2) do ... end
```
and the exported template names will be `foo1` and `foo2`.

I'll make it explicit in the README, because just referring to the extension is confusing in this case.

Reply:
Thanks for the explanation.

Not sure why someone would want to do that, but I understand that it's possible 👍